lib/utils.pm

```diff
@@ -397,6 +398,23 @@ sub sle_version_at_least {
     die "unsupported SLE $version_variable $version in check";
 }
 
+sub install_to_other_at_least {
```

Comment:
I am sorry but the name does not make much sense to me. Care to add a (better) comment on top describing what this is supposed to do? Also, since so far this sub is only used in products/sle/main.pm I recommend to define it just there. We can still move it when we want to use it elsewhere